### ISSUE TITLE
Feedback from 5.1.3 update...

### ISSUE THREAD

**Issue by gaesyeah** [Yesssssman/epicfightmod#651]
"Fixed the vanilla weapon and tools attributes(impact, armor negation, max strikes) not being applied."

-> Great! but there is something more...i tried "epic fight" + "ice and fire" a few days ago and surprisingly EVERYTHING is working well by default!
I dont know who made this, you or the devs from ice and fire, but acctualy the problems with the "tool atributes" still are on with ice and fire weapons/tools. So, if everything from the two mods have compatibility, i think this problem can be called a bug, so im posting here. 

PS: yeah, my english is bad, sorry.

Thanks!

**Comment by Yesssssman** [comment]
Every weapon (Having sweeping edge ability) is applied sword-type capability even though you didn't set both configuration and datapack. And tools do not have a sweeping edge ability, It'll do punching with holding a weapon. To fix this, you must manually set Config file or create Datapack. (p.s. I'm not good at English well!)

**Comment by gaesyeah** [comment]
yeah, i noticed the punching thing with some modded weapons. But acctually EVERY tool from ice and fire are working with epic fight by default, even the hoe(s) and armors(stun armor and weight). I dont know if the dev from ice and fire has changed this because of your mod or all the toes acctualy had "sweeping edge" before purposefully.

-> The only problem is the tooltips, before the last update the vanilla weapons/tools were not showing them, so, if the dev from ice and fire acctualy changed everything i talked about purposefully...maybe he will want to solve this bug that you fixed last patch, i should try to contact him? I opened the issue here because before i didnt had assurance if this (the epic fight tooltips from ice and fire tools/weapons) would be fixable by you or from the ice and fire dev(s)

**Comment by Yesssssman** [comment]
If you're playing on the dedicated server, you have to start the integral server once because Datapack is not supported on the client side. I mean, If you don't load Datapack once, the client side doesn't know the information about the weapons.

**Comment by gaesyeah** [comment]
im acctualy just testing on singleplayer mode. 

-> What is needed to fix this little issue? (yeah i can fix it in the config file. But like i said, the ice and fire dev probably has changed something to every "ice and fire" weapon, tool and armor works 100% with epic fight "system" by default. So, i think it will have more sense if i contact the ice and fire dev directly and explain the situation, right? So, if he wants, he can fix this issue too)

**Comment by gaesyeah** [comment]
I think they have worked on compatibilities some months ago, if you want to try and see, test minecraft with ice and fire + epic fight, you will se that every weapon, tool and armor from ice and fire is already "configured" to work with epic fight by default (in-game), the armor parts for example have very specific values related to "stun armor" and "weight", and seems very balanced in my opinion.

And about my DM, i preffer to talk about in curseforge, because i dont know if you will want to work on the features i talked about there (curse forge), so, i don't want to hype the community with something that might not happen.
->Im gonna send another DM but this time being more "straight to the point".
